microsetta_interface/templates/nutrition.jinja2

```diff
@@ -44,43 +44,39 @@
             preventImplicitSubmission(form_name);
             preclude_whitespace('#ffq_code');
 
-            $("form[name='" + form_name + "']").on('submit', function() {
-                document.getElementById("ffq_code_button").disabled = true;
-            });
+            function handleSubmit() {
+                // TODO go to page since form validated
```

Comment:
This needs to be addressed before we can consider merging the pull request. As it stands, this PR breaks the functionality for the tool.